### PR TITLE
feat: Updated design doc to describe flexible alerting for spending l…

### PR DESCRIPTION
**Pull request by ebadiere**
Updated design doc.
**Related issue(s)**: #2899 

Fixes #

**Notes for reviewer**:

**Checklist**

- [ ] Documented (Code comments, README, etc.)
- [ ] Tested (unit, integration, etc.)

### CODE REVIEW

File: docs/design/hbar-limiter.md [hunk 1]
```diff
@@ -178,6 +184,52 @@ classDiagram
     EthAddressHbarSpendingPlanRepository --> CacheService : uses
     IpAddressHbarSpendingPlanRepository --> CacheService : uses
 ```
+### Support flexible alerting mechanisms for spending thresholds
+The initial spending threshold of the Tier 3 General users will be a rough
```
Comment (quiet-node) [inline docs/design/hbar-limiter.md:188]: I might have missed it but how does alerting system for tier 2 and tier 1 look like? Why is only tier 3 being mentioned here?

File: docs/design/hbar-limiter.md [hunk 2]
```diff
@@ -178,6 +184,52 @@ classDiagram
...
+estimate based on the current daily spending of the Relay Operator.  In order
+to refine this over time a prometheus metric called the `basicSpendingPlanCounter`
+will be used to track the number of unique spending plans.  
+
+The metrics listed below will be added to help determine the best Tier 3 General users
+over time:
+
+1. Daily Unique Users Counter
+2. Average Daily Users
```
Comment (quiet-node) [inline docs/design/hbar-limiter.md:197]: basically, #1 is the total of unique users and #2 is the total users for each day, right?

**Review by quiet-node — APPROVED** [review]

**Review by quiet-node — DISMISSED** [review]

File: docs/design/hbar-limiter.md (hunk 1 above)
Comment (ebadiere) [reply docs/design/hbar-limiter.md:193]: Good point.  It now has more detail.

File: docs/design/hbar-limiter.md (hunk 2 above)
Comment (ebadiere) [reply docs/design/hbar-limiter.md:201]: Yes.

**Review by quiet-node — DISMISSED** [review]

**Review by konstantinabl — APPROVED** [review]

**Review by quiet-node — APPROVED** [review]

**Pull request merged**